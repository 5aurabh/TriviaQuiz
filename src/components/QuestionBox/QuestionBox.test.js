import React from 'react';
import renderer from 'react-test-renderer';
import Questionbox from '.';

it('QuestionBox: default', () => {
  const component = renderer.create(
    <Questionbox question={{ category: 's', question: 'a' }} />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
