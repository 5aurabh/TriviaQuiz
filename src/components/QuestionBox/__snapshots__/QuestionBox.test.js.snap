// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionBox: default 1`] = `
<div
  className={undefined}
>
  <div
    className={undefined}
  >
    s
  </div>
  <div
    className={undefined}
  >
    <div
      className={undefined}
    >
      a
    </div>
    <div
      className={undefined}
    >
      <div>
        <div
          className={undefined}
        >
          <input
            className={undefined}
            defaultChecked={null}
            id="question-sol-true"
            name="question-sol"
            onChange={[Function]}
            type="radio"
            value={true}
          />
          <label
            className={undefined}
            htmlFor="question-sol-true"
          >
            True
          </label>
        </div>
      </div>
      <div>
        <div
          className={undefined}
        >
          <input
            className={undefined}
            defaultChecked={null}
            id="question-sol-false"
            name="question-sol"
            onChange={[Function]}
            type="radio"
            value={false}
          />
          <label
            className={undefined}
            htmlFor="question-sol-false"
          >
            False
          </label>
        </div>
      </div>
      <div
        className={undefined}
      >
        <button
          className="undefined undefined"
          disabled={true}
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "grey",
              "color": "#fff",
              "cursor": "not-allowed",
            }
          }
        >
          Submit
        </button>
      </div>
    </div>
  </div>
</div>
`;
